docker/gcp_pudl_etl.sh

```diff
@@ -95,17 +95,23 @@ ETL_SUCCESS=${PIPESTATUS[0]}
 
 # if pipeline is successful, distribute + publish datasette
 if [[ $ETL_SUCCESS == 0 ]]; then
-    # Dump outputs to s3 bucket if branch is dev or build was triggered by a tag
-    if [ $GITHUB_ACTION_TRIGGER = "push" ] || [ $GITHUB_REF = "dev" ]; then
-        copy_outputs_to_distribution_bucket
-        ETL_SUCCESS=${PIPESTATUS[0]}
-    fi
-
     # Deploy the updated data to datasette
     if [ $GITHUB_REF = "dev" ]; then
         python ~/devtools/datasette/publish.py 2>&1 | tee -a $LOGFILE
         ETL_SUCCESS=${PIPESTATUS[0]}
     fi
+
+    # Compress the SQLite DBs for easier distribution
+    gzip $PUDL_OUTPUT/*.sqlite
+
+    # Remove redundant multi-file EPA CEMS outputs prior to distribution
+    rm -rf $PUDL_OUTPUT/hourly_emissions_epacems/
```

Comment:
We have to generate the partitioned EPA CEMS outputs during the ETL since we do it all in parallel, but for distribution, right now I think we can just push out the one big file, which will be simpler to understand, smaller, follow the same pattern as the other Parquet files. If we've put the right metadata into the Parquet file then tools that understand Parquet shouldn't need to download a bunch of information to work with the file remotely.

Reply:
Updated to produce a bit of logging output with `gzip` and to affect the success of the overall deployment.